backend/plugin/db/mysql/dump.go

```diff
@@ -622,12 +626,17 @@ func isNumeric(t string) bool {
 }
 
 // getRoutines gets all routines of a database.
-func getRoutines(txn *sql.Tx, dbName string) ([]*routineSchema, error) {
+func getRoutines(txn *sql.Tx, dbType db.Type, dbName string) ([]*routineSchema, error) {
 	var routines []*routineSchema
 	for _, routineType := range []string{"FUNCTION", "PROCEDURE"} {
 		if err := func() error {
-			query := fmt.Sprintf("SHOW %s STATUS WHERE Db = ?;", routineType)
-			rows, err := txn.Query(query, dbName)
+			var query string
+			if dbType == "OCEANBASE" {
+				query = fmt.Sprintf("SHOW %s STATUS FROM %s;", routineType, dbName)
+			} else {
+				query = fmt.Sprintf("SHOW %s STATUS WHERE Db = %s;", routineType, dbName)
```

Comment:
--- FAIL: TestGetLatestSchema/MySQL

cd backend/tests
go test --tags=mysql -v -run TestGetLatestSchema

failed to get routines of database \"latestSchema\

Reply:
> --- FAIL: TestGetLatestSchema/MySQL
> 
> cd backend/tests go test --tags=mysql -v -run TestGetLatestSchema
> 
> failed to get routines of database "latestSchema\

Seems need to add single quotes here in the where clause, modified and verified locally.